Modix.Bot/Modules/InfractionModule.cs

```diff
@@ -98,10 +98,13 @@ public InfractionModule(IModerationService moderationService, IUserService userS
 
         [Command("delete")]
         [Summary("Marks an infraction as deleted, so it no longer appears within infraction search results")]
-        public Task Delete(
+        public async Task Delete(
             [Summary("The ID value of the infraction to be deleted.")]
                 long infractionId)
-            => ModerationService.DeleteInfractionAsync(infractionId);
+        {
+            await ModerationService.DeleteInfractionAsync(infractionId);
+            await Context.Message.AddReactionAsync(new Emoji("✅"));
```

Comment:
Just wondering out loud... Does it make sense for this to be an extension method, maybe on `ModuleBase`? Granted, it's a fairly simple one-liner, so maybe not, but I do see that this pattern is already used in a number of places.

Reply:
I had the same thought, and ended up implementing it as an extension method on ICommandContext in a local branch - I'll leave it to @jrusbatch whether he wants to bother for this PR.